### DISCUSSION TITLE
Mupif repository and prerequisites on Musicode-dedicated machine

### DISCUSSION THREAD

**Discussion by solidkon** [mupif/mupif#63] [Q&A] [answered]
Hi!

I want to start trying the DemoModel() class on the pc204 that is dedicated to test Musicode-related activities. Do you need this machine for certain special tests or we can install a fresh anaconda distribution there and the latest mupif repository? Currently I cannot run anything there that imports mupif.

**Comment by eudoxos** [comment, marked as answer]
As far as I know, we don't need the machine for ourselves. The normal tests are running on our own nodes in the VPN (plus @stanislavsulc is sometimes accessing the cluster).  So go ahead. (I hope this was a question for out team ;) If not, tag the addressees with @ ).

**Reply by solidkon** [reply]
yes thanks a lot! It was for your team
